test/files/run/compiler-asSeenFrom.scala

```diff
@@ -1,56 +1,10 @@
 /*
  * filter: inliner warning; re-run with
```

Comment:
The unmodified test succeeds without this filter.

Reply:
Done.

Time for a partest front matter linter?  😄